src/client/datascience/constants.ts

```diff
@@ -379,7 +379,6 @@ export enum Telemetry {
     DebugpyInstallFailed = 'DATASCIENCE.DEBUGPY_INSTALL_FAILED',
     DebugpyInstallCancelled = 'DATASCIENCE.DEBUGPY_INSTALL_CANCELLED',
     ScrolledToCell = 'DATASCIENCE.SCROLLED_TO_CELL',
-    ExecuteNativeCell = 'DATASCIENCE.NATIVE.EXECUTE_NATIVE_CELL',
```

Comment:
This is wrong, interactive also uses this event, hence the data is already kaput.
We have a better way of determining whether its executing via Native notebooks